be/src/vec/common/arena.h

```diff
@@ -127,11 +127,16 @@ class Arena : private boost::noncopyable {
 
 public:
     Arena(size_t initial_size_ = 4096, size_t growth_factor_ = 2,
-          size_t linear_growth_threshold_ = 128 * 1024 * 1024)
+          size_t linear_growth_threshold_ = -1)
             : growth_factor(growth_factor_),
-              linear_growth_threshold(linear_growth_threshold_),
               head(new Chunk(initial_size_, nullptr)),
-              size_in_bytes(head->size()) {}
+              size_in_bytes(head->size()) {
+        if (linear_growth_threshold_ < 0) {
+            linear_growth_threshold = config::memory_linear_growth_threshold;
```

Comment:
No, so I just keep the default value to 128Mb, the same with the previous value.
It needs to be a config, because we have also have mempool, they should use the same config.